docs/nuget-org/TOC.md

```diff
@@ -7,6 +7,7 @@
 ## [Publish a package](publish-a-package.md)
 ## [Package ID prefix reservation](id-prefix-reservation.md)
 ## [Package deprecation](deprecate-packages.md)
+## [Package Readme](package-readme-on-nuget-org.md)
```

Comment:
Lowercase `readme` would be more consistent with the other TOC entries?

Reply:
README imo to be consistent.